Framework/DataHandling/src/LoadMcStasNexus.cpp

```diff
@@ -42,10 +43,19 @@ const std::string LoadMcStasNexus::category() const { return "DataHandling\\Nexu
  * @returns An integer specifying the confidence level. 0 indicates it will not
  * be used
  */
-int LoadMcStasNexus::confidence(Kernel::NexusDescriptor &descriptor) const {
-  UNUSED_ARG(descriptor)
-  // To ensure that this loader is somewhat hitten return 0
+int LoadMcStasNexus::confidence(Kernel::NexusHDF5Descriptor &descriptor) const {
   int confidence(0);
+  const auto entries = descriptor.getAllEntries();
```

Comment:
Can we capture this by reference to avoid the copy?

Reply:
Yes absolutely. I thought I had done that.